jsonnet/utils/sanitize-rules.libsonnet

```diff
@@ -73,6 +74,26 @@ local excludedRules = [
       { alert: 'PrometheusErrorSendingAlertsToAnyAlertmanager' },
     ],
   },
+  {
+    name: 'kube-prometheus-node-recording.rules',
+    rules: [
+      { record: 'instance:node_cpu:ratio' },
+    ],
+  },
+  {
+    name: 'node.rules',
+    rules: [
+      { record: 'node:node_num_cpu:sum' },
+    ],
+  },
+  {
+    name: 'openshift-kubernetes.rules',
+    rules: [
+      { record: 'namespace:container_spec_cpu_shares:sum' },
+      { record: 'pod:container_memory_usage_bytes:sum' },
+      { record: 'pod:container_spec_cpu_shares:sum' },
+    ],
+  },
```

Comment:
can you add a comment with a link to the BZ so we don't have to look into the Git history to understand why these rules are removed?